### PR TITLE
Add Apache 2 license headers to Android sample code.

### PR DESCRIPTION
**Pull request by tswast**
Also, renames the package to match the reverse domain name conventions.

### CODE REVIEW

**Comment by amygdala** [comment]
that lgtm.  Thanks!  I assume the version change in the android tools package has no impact?

**Comment by tswast** [comment]
Yeah, the Android Studio gave me a warning that the plugin we had was too old and it auto-updated it to this. Was an expired alpha version, I think.

On the plus side, that change might fix the Android build on Travis. Fingers crossed for that, though I did test on my phone.

File: android/CloudVision/app/src/main/java/com/google/sample/cloudvision/MainActivity.java [hunk 1]
```diff
@@ -130,7 +153,10 @@ public void uploadImage(Uri uri) {
         if (uri != null) {
             try {
                 // scale the image to 800px to save on bandwidth
-                Bitmap bitmap = scaleBitmapDown(MediaStore.Images.Media.getBitmap(getContentResolver(), uri), 1200);
+                Bitmap bitmap =
+                        scaleBitmapDown(
+                                MediaStore.Images.Media.getBitmap(getContentResolver(), uri),
+                                1200);
```
Comment (benwicks) [inline android/CloudVision/app/src/main/java/com/google/sample/cloudvision/MainActivity.java]: Why is this `maxDimen` parameter 1200 but the comment on L#155 says 800?

Comment (tswast) [inline android/CloudVision/app/src/main/java/com/google/sample/cloudvision/MainActivity.java]: /shruggie I'm not sure. I think it was originally set to 800. I'm not sure in which PR it was updated.

**Pull request merged**